### ISSUE TITLE
Subscribe method isn't working on the latest firebase_cpp_sdk (10.0.0) 

### ISSUE THREAD

**Issue by DavidCaminal** [Larpon/QtFirebase#174]
Well, as i mentioned in the title, the subscribe method doesn't seem to work with this sdk. It doesn't subscribe the device nor does it output an error as expected. 
I've tested this both in my own project and on the [QtFirebaseExample](https://github.com/Larpon/QtFirebaseExample) documented here. I've already tested the latest tested sdk version (6.7.0) and it indeed works. 
I was wondering. Do you have any clue why 10.0.0 might not work with this project?
Alternatively, how much functionality would I be missing by settling for the older 6.7.0 sdk version?

**Comment by Larpon** [comment]
> I was wondering. Do you have any clue why 10.0.0 might not work with this project?

Not really, no. I can only guess that it is Google's constant "beta" state that they're releasing, more or less, all their products under.
The APIs I've battled to support in QtFirebase's lifetime has almost always been broken between updates - so `10.0.0` not working is not really a surprise to me. That said; I'm pretty sure QtFirebase can be brought up to speed fairly quickly. But it'll have to be someone from the small community here that does it, since I have long moved away from Firebase.

> Alternatively, how much functionality would I be missing by settling for the older 6.7.0 sdk version?

That's hard to say, judging from the readme there's no new major sub-systems added. Looking at how Google (and Apple - in regards to the iOS version of Firebase) constantly evolve their software, without much attention to backwards compatibility, my guess is that you'll sooner - rather than later - experience some kind of breakage with a version so many commits away. But I am no oracle... so your guess is as good as mine :slightly_smiling_face:
